src/monodroid/jni/monodroid-glue.cc

```diff
@@ -1759,6 +1759,29 @@ MonodroidRuntime::Java_mono_android_Runtime_destroyContexts (JNIEnv *env, jclass
 	log_info (LOG_DEFAULT, "All domain cleaned up");
 }
 
+const char*
```

Comment:
This should have a `char*` return type, not `const char*`, because the returned string should be free'd by the caller.